docs: Add description for NEW_RELIC_CLOUD_AWS_ACCOUNT_ID (#21740)
newrelic/docs-website: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/env-variables-lambda.mdx

                </ul>
            </td>
        </tr>
        <tr>
            <td>`NEW_RELIC_CLOUD_AWS_ACCOUNT_ID`</td>
            <td></td>
            <td></td>
            <td>

>                Provide the AWS Account ID where your monitored resources (e.g., databases, Lambda functions) are located. This allows New Relic to correctly map and display relationships between these monitored entities.

Reviewer: ```suggestion
               To help New Relic correctly map and display relationships between your monitored entities, provide the AWS Account ID where your monitored resources, such as databases and Lambda functions.
```
